nat-instance.tf

```diff
@@ -45,6 +47,17 @@ resource "aws_security_group_rule" "nat_instance_ingress" {
   type              = "ingress"
 }
 
+resource "aws_security_group_rule" "nat_instance_ssh_ingress" {
+  count             = local.enabled ? local.nat_instance_enabled : 0
+  description       = "Allow ingress traffic for ssh"
+  from_port         = 22
+  to_port           = 22
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = join("", aws_security_group.nat_instance.*.id)
+  type              = "ingress"
+}
+
```

Comment:
If this is desired, this can be attached outside of this module. I don't think we should allow ssh when we have ssm.

```suggestion
```